aloha-io-avro/src/main/scala/com/eharmony/aloha/factory/avro/StdAvroModelFactory.scala

```diff
@@ -1,27 +1,35 @@
 package com.eharmony.aloha.factory.avro
 
-import java.io.{File, InputStream}
+import java.io.File
 
-import com.eharmony.aloha.audit.impl.avro.{AvroScoreAuditor, Score}
+import com.eharmony.aloha.audit.impl.avro.Score
 import com.eharmony.aloha.factory.ModelFactory
-import com.eharmony.aloha.factory.ex.AlohaFactoryException
-import com.eharmony.aloha.reflect.{RefInfo, RefInfoOps}
-import com.eharmony.aloha.semantics.compiled.CompiledSemantics
-import com.eharmony.aloha.semantics.compiled.compiler.TwitterEvalCompiler
-import com.eharmony.aloha.semantics.compiled.plugin.avro.CompiledSemanticsAvroPlugin
-import org.apache.avro.Schema
 import org.apache.avro.generic.GenericRecord
-import org.apache.commons.io.IOUtils
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.util.{Failure, Success, Try}
+import scala.util.Try
+
+
 
 /**
   * Created by deak on 3/2/17.
   */
 object StdAvroModelFactory {
 
   /**
+    * Provides a standard way to create Avro ModelFactory instances for producing
+    * models that take `GenericRecord`s and return `Score`s.
+    * @param conf a factory configuration.  This can also be used via the Magnet
+    *             pattern.  See implicits in the [[FactoryConfig]] companion object
+    *             to see possible parameter combinations.  Also see the Spray
+    *             [[http://spray.io/blog/2012-12-13-the-magnet-pattern/ Magnet pattern]]
+    *             article for more information.  To enable Magnet pattern syntax, see
+    *             [[FactoryConfig.MagnetSyntax]]
+    *
+    * @return a Try of a ModelFactory.
+    */
+  def fromConfig(conf: FactoryConfig): Try[ModelFactory[GenericRecord, Score]] = conf()
```

Comment:
I like this.  I wish this could be the apply method but it can't because of the previously defined one.  This defines the possibility of the Magnet pattern, which I like.